agent/exec/container/executor.go

```diff
@@ -76,6 +76,7 @@ func (e *executor) Describe(ctx context.Context) (*api.NodeDescription, error) {
 			NanoCPUs:    int64(info.NCPU) * 1e9,
 			MemoryBytes: info.MemTotal,
 		},
+		Addr: info.Swarm.NodeAddr,
```

Comment:
Honestly I have no idea where that `Swarm.NodeAddr` field comes from. TBH. I'm trying to figure that out right now.